### PR TITLE
my-daily-norma-and-modal

### PR DESCRIPTION
**Pull request by AllaPavlovska**

### CODE REVIEW

File: src/components/MyDailyNorma/MyDailyNorma.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import styles from './MyDailyNorma.module.css';
```
Comment (Kateryna-Borysenko) [inline src/components/MyDailyNorma/MyDailyNorma.jsx:2]: pls make a replacement
from 'styles' to 's' we should stick to one style code

File: src/components/MyDailyNorma/MyDailyNormaModal.jsx [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    >
+      <div className={styles.modalContainer} onClick={e => e.stopPropagation()}>
+        <div
+          style={{
+            display: 'flex',
+            justifyContent: 'space-between',
+            alignItems: 'center',
+          }}
+        >
+          <h2 className={styles.modalNormaTitle}>My daily norma</h2>
```
Comment (Kateryna-Borysenko) [inline src/components/MyDailyNorma/MyDailyNormaModal.jsx:41]: use reusable Title component  components/common/Title/Title

File: src/components/MyDailyNorma/MyDailyNormaModal.jsx [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+                    <Field
+                      className={styles.modalInput}
+                      type="text"
+                      name="drankWater"
+                      placeholder="0"
+                    />
+                  </label>
+                </div>
+                <div className={styles.divButtonModal}>
+                  <button className={styles.ModalButton} type="submit">
```
Comment (Kateryna-Borysenko) [inline src/components/MyDailyNorma/MyDailyNormaModal.jsx:151]: use reusable Button component    uikit/Button/Button

File: src/components/MyDailyNorma/MyDailyNormaModal.module.css [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+  border-radius: 10px;
+  width: 592px;
+  height: 647px;
+}
+
+.modalNormaTitle {
+  margin-top: 0;
+  width: 180px;
+}
+.closeIcon {
```
Comment (Kateryna-Borysenko) [inline src/components/MyDailyNorma/MyDailyNormaModal.module.css:25]: space

File: src/components/MyDailyNorma/MyDailyNormaModal.module.css [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+  color: var(--primary-sky-blue);
+}
+
+.modalParagraphContainer {
+  margin-top: 24px;
+  margin-bottom: 24px;
+}
+.secondContainer {
```
Comment (Kateryna-Borysenko) [inline src/components/MyDailyNorma/MyDailyNormaModal.module.css:33]: space

File: src/components/MyDailyNorma/MyDailyNormaModal.module.css [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+  display: flex;
+}
+.highlight {
```
Comment (Kateryna-Borysenko) [inline src/components/MyDailyNorma/MyDailyNormaModal.module.css:36]: check all spaces

File: src/pages/WelcomePage/WelcomePage.jsx [hunk 7]
```diff
@@ -9,6 +10,7 @@ const WelcomePage = () => {
 
       <Container>
         <div>Welcome to Water Tracker App</div>
+        <MyDailyNorma />
```
Comment (Kateryna-Borysenko) [inline src/pages/WelcomePage/WelcomePage.jsx:13]: delete pls, we'll import this component later in the right way

File: src/components/MyDailyNorma/MyDailyNormaModal.jsx [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+            </p>
+          </div>
+          <p className={styles.modalP}>
+            * V is the volume of the water norm in liters per day, M is your
+            body weight, T is the time of active sports, or another type of
+            activity commensurate in terms of loads (in the absence of these,
+            you must set 0)
+          </p>
+
+          <h3 className={styles.modalNormaText}>Calculate your rate:</h3>
```
Comment (Kateryna-Borysenko) [inline src/components/MyDailyNorma/MyDailyNormaModal.jsx:78]: use Subtitle component

File: src/components/MyDailyNorma/MyDailyNormaModal.module.css [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+  color: var(--primary-sky-blue);
+}
+.ModalButton:hover {
+  background-color: #0056b3;
+}
+.divButtonModal {
+  display: flex;
+  justify-content: flex-end;
+  margin-top: 24px;
+}
```
Comment (Kateryna-Borysenko) [inline src/components/MyDailyNorma/MyDailyNormaModal.module.css:107]: We are developing our app according to the mobile-first principle, is the component adapted?

File: src/components/MyDailyNorma/MyDailyNormaModal.jsx [hunk 10]
```diff
@@ -0,0 +1,164 @@
...
+  };
+
+  const handleKeyPress = event => {
+    if (event.key === 'Escape') {
+      closeModal();
+    }
+  };
+
+  return (
+    <div
```
Comment (GermanSTr) [inline src/components/MyDailyNorma/MyDailyNormaModal.jsx:28]: Є компонент модалки, який необхідно використати,  src/components/common/Modal

File: src/components/MyDailyNorma/MyDailyNormaModal.jsx [hunk 11]
```diff
@@ -0,0 +1,164 @@
...
+      onClick={closeModal}
+      onKeyDown={handleKeyPress}
+    >
+      <div className={styles.modalContainer} onClick={e => e.stopPropagation()}>
+        <div
+          style={{
+            display: 'flex',
+            justifyContent: 'space-between',
+            alignItems: 'center',
+          }}
```
Comment (GermanSTr) [inline src/components/MyDailyNorma/MyDailyNormaModal.jsx:39]: Стилі прописуємо в module.css

File: src/components/MyDailyNorma/MyDailyNorma.jsx [hunk 12]
```diff
@@ -0,0 +1,16 @@
+import MyDailyNormaButton from './MyDailyNormaButton';
+import s from './MyDailyNorma.module.css';
+
+const MyDailyNorma = () => {
+  return (
+    <div className={s.dailyNormaContainer}>
+      <h2 className={s.dailyNormaTitle}>My daily norma</h2>
```
Comment (GermanSTr) [inline src/components/MyDailyNorma/MyDailyNorma.jsx:7]: Тут можна компонент Title використати

**Review by HalakII — APPROVED** [review]

File: src/components/MyDailyNorma/MyDailyNormaButton.jsx [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+export const MyDailyNormaButton = () => {
+  const [isOpenModal, setIsOpenModal] = useState(false);
+
+  const handleCloseModal = () => {
+    setIsOpenModal(prevState => !prevState);
+  };
+
+  return (
+    <>
+      <button
```
Comment (GermanSTr) [inline src/components/MyDailyNorma/MyDailyNormaButton.jsx:15]: тут компонент Button можно використати

**Review by Kateryna-Borysenko — APPROVED** [review]

**Pull request merged**